LION/data_loaders/deteCT.py

```diff
@@ -380,14 +381,13 @@ def __load_and_preprocess_segmentation__(self, index):
         return segmentation
 
     def __getitem__(self, index):
-
         index = int(index)  # cast to int
 
         # If input is sinogram, we need to load the sinogram
         if self.task in ["sino2sino", "sino2recon", "sino2seg", "joint"]:
             input = self.__load_and_preprocess_sinogram__(index, self.input_mode)
         # if input is reconstruction, we need to load the reconstruction
-        else:
+        elif self.task != "groundtruth":
```

Comment:
This should still just be an else. 

The GT of 2DeteCT is a recon done by Max, so even if you only want GT, you may want this GT to be reconstructed by you.

Reply:
That makes sense, it should be fixed by 27f5bab.